lib/wallet/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -1589,22 +1574,16 @@ mkTxSkeleton witness context skeleton = TxSkeleton
     , txAssetsToMintOrBurn = (<>)
         (TokenMap.getAssets (fst $ view #txAssetsToMint context))
         (TokenMap.getAssets (fst $ view #txAssetsToBurn context))
-    , txScriptExecutionCost = view #txPlutusScriptExecutionCost context
     }
 
 -- | Estimates the final cost of a transaction based on its skeleton.
 --
-estimateTxCost :: AnyCardanoEra -> ProtocolParameters -> TxSkeleton -> Coin
-estimateTxCost era pp skeleton =
-    F.fold
-        [ computeFee (estimateTxSize era skeleton)
-        , view #txScriptExecutionCost skeleton
-        ]
+estimateTxCost :: AnyCardanoEra -> FeePerByte -> TxSkeleton -> Coin
+estimateTxCost era (FeePerByte feePerByte) skeleton =
+    computeFee (estimateTxSize era skeleton)
   where
     computeFee :: TxSize -> Coin
-    computeFee (TxSize size) =
-        let LinearFee LinearFunction {..} = getFeePolicy $ txParameters pp
-        in Coin $ ceiling $ intercept + slope * fromIntegral size
```

Comment:
💀 right, can't drop `intercept` like this when migration is relying it as well.

Reply:
I think it would be easier to deal with after https://github.com/input-output-hk/cardano-wallet/pull/3893

Reply:
Comment on solution in commits message: https://github.com/input-output-hk/cardano-wallet/pull/3894/commits/7ca0fa10109da461ffe16682ca3f2782430669b5

Reply:
Needed to remove some tests as well: https://github.com/input-output-hk/cardano-wallet/pull/3894/commits/ee5f851832927882db3a8010640775ab6680177f